spec/classes/redis_spec.rb

```diff
@@ -99,6 +108,7 @@ class { 'redis::globals':
 
       context 'with ulimit' do
         let(:params) { { ulimit: 7777 } }
+        let(:facts) { super().merge(puppetversion: Puppet.version) }
```

Comment:
I thought rspec-puppet-facts did this for us:
https://github.com/mcanevet/rspec-puppet-facts/blob/e02496555b0a79c0c0434ed99557b7097c5771b4/lib/rspec-puppet-facts.rb#L237

Reply:
Good catch, fixed in most recent commit.